eng/pipelines/common/variables.yml

```diff
@@ -7,6 +7,8 @@ variables:
   value: true
 - name: DOTNET_VERSION
   value: 5.0.201
+- name: XHARNESS_VERSION
+  value: 1.0.0-prerelease.21307.1
```

Comment:
This is added because we need to update to the latest xharness to use with xcode 12.5, but that now requires .net 6. Until we get net6 everywhere, we can use the old xharness on the non-net6 machines.